tests/test_cli.py

```diff
@@ -353,7 +367,188 @@ def test_mapping_to_jinja():
 
     result = runner.invoke(
         mapping_to_jinja_cli,
-        ['-', '-', '--type', type_name],
+        ['-', '-', '--es_type', type_name],
+        input=json.dumps(mapping),
+    )
+
+    assert_no_exception(result)
+    # use the child template to modify the output template.
+    jinja_env = jinja2.Environment(
+        loader=jinja2.DictLoader({'generated': result.output}))
+    gen_output = json.loads(jinja_env.from_string(child_template).render())
+    assert gen_output == expected
+
+
+def test_multiple_mapping_to_jinja():
+    """Test mapping_to_jinja."""
+    mapping = {
+        "mappings": {
+            "test1": {
+                "properties": {
+                    'sub1': {
+                        'type1': 'string',
+                    }
+                }
+            },
+            "test2": {
+                "properties": {
+                    'sub2': {
+                        'type2': 'string',
+                    }
+                }
+            }
+        }
+    }
+
+    # template which extends the generated one and will be compiled.
+    child_template = """
+    {% extends "generated"%}
```

Comment:
Could the test try to modify the mapping of each type? That way we can check that the jinja blocks are properly generated. Something similar to the test with only one type.
Otherwise the test won't detect if something is broken in the jinja block name generation.